validator/testcases/content.py

```diff
@@ -297,13 +298,13 @@ def _process_file(err, xpi_package, name, file_data, name_lower,
         try:
             sub_xpi = XPIManager(package, mode="r", name=name,
                                  subpackage=is_subpackage)
-        except Exception:
+        except BadZipfile:
```

Comment:
We should probably move this test and reporting to `XPIManager`, but we're not testing the same file twice. In the other instance, we're testing a top-level add-on package, where this is testing a JAR with arbitrary content that belongs to the same add-on.